docs/docs/jupyter-integration.ipynb

```diff
@@ -0,0 +1,334 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "3ea7576e-1f54-4983-a8d1-1f6c0e2165de",
+   "metadata": {},
+   "source": [
+    "# Jupyter Integration\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d5d9b03a-627f-44d1-87da-caf2597f29dd",
+   "metadata": {},
+   "source": [
+    "Datapane has first-class Jupyter Notebook support. This includes Jupyter Lab, Jupyter Notebook, Google Colab, and Visual Studio Code with the Jupyter plugin.\n",
+    "\n",
+    "Let's see how Datapane and Jupyter Notebook can build data-powered apps, all without disrupting your workflow."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e8275e34-e775-469f-b53a-a4e9bf5ed616",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import datapane as dp\n",
+    "import altair as alt\n",
+    "from vega_datasets import data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b59bc5e6-a941-4b03-ba16-6734ec57e740",
+   "metadata": {},
+   "source": [
+    "## Notebook to Data App? One line of code."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "999f4dd5-b061-4eab-bde9-42db08790265",
+   "metadata": {},
+   "source": [
+    "Data analysis often begins in a Jupyter Noteboos, and once complete, we need to share our insights.\n",
+    "\n",
+    "Some recipients may be comfortable with an `.ipynb` file, and in those instances, you could just send your notebook over and still benefit from Datapane's enhancements.\n",
+    "\n",
+    "But in most cases, you will want to turn your notebook into something presentable and accessible. With Datapane, all it takes is a single line of code.\n",
+    "\n",
+    "```python\n",
+    "app = dp.App(blocks=dp.cells_to_blocks(Out))\n",
+    "```\n",
+    "\n",
+    "Datapane will automatically detect your cells and convert them to Datapane blocks.\n",
+    "\n",
+    "\n",
+    "![Iris analysis to app](img/jupyter/iris-analysis.png)\n",
```

Comment:
I **really** love this image, but do you have a higher DPI version?

Reply:
I didn't as it was a quick one - but I've just remade it! Looking crisp now. Glad you like it!